docs/help.md

```diff
@@ -0,0 +1,150 @@
+   - **Brevity**: Usage output must be concise and limited to the essential syntax.
+   - **Exit Code**: Displaying usage must result in a **non-zero exit code**.
+   - **No Logo**: The usage output must never include the Atmos logo or branding.
+
+3. **Output Characteristics**:
+   - Usage must clearly indicate:
+     - Command syntax.
+     - Required and optional arguments.
+     - Minimal explanation to guide correction of misuse.
+
+---
+
+## Help Guidelines
+
+1. **When to Display Help**:
+   - Display when the user explicitly requests it, such as:
+     - Running the `help` subcommand:
+
+       ```bash
+       atmos help
+       ```
+
+     - Using the `--help` flag with a command:
+
+       ```bash
+       atmos terraform --help
+       ```
+
+2. **Behavior**:
+   - **Comprehensive**: Help output must include:
+     - A description of the command.
+     - All available options, flags, and parameters.
+     - Examples of usage.
+   - **Error-Free**:
+     - Help must always succeed and exit with a **zero exit code**.
+     - Help should not fail due to misconfigured files, missing dependencies, or validation errors.
+   - **Static**:
+     - Help must be non-interactive and preformatted.
+
+3. **Atmos Logo**:
+   - The Atmos logo can be included in the `help` output, especially for the main `help` command.
+   - Subcommands or options help may omit the logo to keep focus on the content.
+
+4. **No Validation**:
+   - Help must not perform any validation, such as:
+     - Checking configuration files.
+     - Verifying that required tools (e.g., Terraform) are installed.
+
+5. **Example Output**
+
+   ```plaintext
+   Atmos CLI - Version 1.0.0
+
+   Usage:
+       atmos <command> [options]
+
+   Available Commands:
+       terraform    Manage Terraform workflows
+       helmfile     Manage Helmfile workflows
+       help         Show this help message
+
+   Options:
+       -v, --version    Show the Atmos version
+       -h, --help       Show help for a command
+
+   Examples:
+
+   - Display the current atmos version
+
+     $ atmos version
+
+   - Display the help for the terraform subcommand
+
+     $ atmos terraform --help
+
+   ```
+
+---
+
+## Comparison of Usage and Help
+
+| Feature                 | Usage                                   | Help                               |
+| ----------------------- | --------------------------------------- | ---------------------------------- |
+| **Purpose**             | Quick syntax guidance after an error    | Comprehensive documentation        |
+| **When Displayed**      | After a command misuse or error         | Explicitly requested by the user   |
+| **Content**             | Syntax and required arguments only      | Detailed descriptions and examples |
+| **Exit Code**           | Non-zero (when accompanied by an error) | Zero                               |
+| **Includes Logo**       | No                                      | Yes (optional for subcommands)     |
```

Comment:
I would suggest ATMOS Logo should either be present in all command helps or none.  Optional for subcommands bothers me.

Reply:
Fine by me